js/app/controllers/attendeecontroller.js

```diff
@@ -114,7 +115,8 @@ app.controller('AttendeeController', function($scope, AutoCompletionService) {
 				role: 'REQ-PARTICIPANT',
 				rsvp: 'TRUE',
 				partstat: 'NEEDS-ACTION',
-				cutype: 'INDIVIDUAL'
+				cutype: 'INDIVIDUAL',
+				isregistered: item.is_local_user ? 'TRUE' : 'FALSE'
```

Comment:
You can use `true` and `false` here. 
RSVP is a string, because that item is part of the calendar data and encoded as a string there